### PR TITLE
updating raid check to alert at 300 media errors

### PR DESCRIPTION
**Pull request by aacole**

### CODE REVIEW

File: sensu/plugins/check-raid.sh [hunk 1]
```diff
@@ -12,9 +12,9 @@ if lspci | grep RAID | grep -i 3ware >> /dev/null; then
     sudo check_3ware_raid.py -b /usr/sbin/tw-cli -z $CRITICALITY
 elif lspci | grep RAID | grep -i "MegaRAID" >> /dev/null; then
     if [[ -e /etc/sensu/plugins/check-storcli.pl && -e /opt/MegaRAID/storcli/storcli64 ]];then
-      sudo check-storcli.pl -p /opt/MegaRAID/storcli/storcli64 -Io 63 -z $CRITICALITY
+      sudo check-storcli.pl -p /opt/MegaRAID/storcli/storcli64 -Io 63 -Im 300 -z $CRITICALITY
     else
-      sudo check_megaraid_sas.pl -b /usr/sbin/megacli -o 63 -z $CRITICALITY
+      sudo check_megaraid_sas.pl -b /usr/sbin/megacli -o 63 -m 300 -z $CRITICALITY
```
Comment (pgraziano) [inline sensu/plugins/check-raid.sh:17]: should this be `-Im 300`?

Comment (pgraziano) [reply sensu/plugins/check-raid.sh:17]: my bad, `-m 300` is correct.  This looks good.

Comment (aacole) [reply sensu/plugins/check-raid.sh:17]: the check_megaraid_sas.pl script has a slightly different syntax

```
$ ./check_megaraid_sas.pl -h
Usage: ./check_megaraid_sas.pl [-s number] [-m number] [-o number] [-z criticality]
       -b the megacli binary path to use
       -z the criticality of this alert
	      -e run megacli with sudo privileges       -s is how many hotspares are attached to the controller
       -m is the number of media errors to ignore
       -p is the predictive error count to ignore
       -o is the number of other disk errors to ignore
```

**Review by pgraziano — APPROVED** [review]
lgtm

**Pull request merged**